File: array_problems/findMissingElement/findMissingElement.php
<?php
class ArrayProblems {
    public function findMissingElement($arr) {
        $missings = array();
        for ($i = min($arr); $i <= max($arr); $i++) {
            array_push($missings, $i);
        }
        $filtered_missings = array_filter($missings, function($i) use ($arr) {
            return !in_array($i, $arr);
        });
        $filtered_str = implode(',', $filtered_missings);
        $filtered_result = array_map('intval', explode(',', $filtered_str));
        return $filtered_str;
    }
}
$array_problems = new ArrayProblems();
$missingElement = $array_problems->findMissingElement([1, 2, 3, 5, 6, 7, 8, 9, 11, 12, 13, 14, 15, 16, 17, 18, 20, 21, 22, 23, 24, 25, 26, 27, 29, 30, 31, 32, 33, 34, 35, 37, 38, 39, 40, 41, 42, 43, 45, 46, 47, 48, 49, 50]);
echo($missingElement);

File: array_problems/findMissingElement/findMissingElement.py
class ArrayProblems:
    def findMissingElement(self, arr):
        missings = []
        for i in range(min(arr), max(arr) + 1):
            missings.append(i)
        filtered_missings = [int(i) for i in missings if int(i) not in arr]
        filtered_result = list(map(int, filtered_missings))
        return filtered_result


array_problems = ArrayProblems()
missing_element = array_problems.findMissingElement([1, 2, 3, 5, 6, 7, 8, 9, 11, 12, 13, 14, 15, 16, 17, 18, 20, 21, 22, 23, 24, 25, 26, 27, 29, 30, 31, 32, 33, 34, 35, 37, 38, 39, 40, 41, 42, 43, 45, 46, 47, 48, 49, 50])
print(missing_element)